lib/ansible/modules/system/firewalld.py

```diff
@@ -98,6 +98,26 @@
       - Whether to run this module even when firewalld is offline.
     type: bool
     version_added: "2.3"
+  direct_rule:
+    description:
+      - 'The firewalld iptables direct rule you would like to enable/disable.'
+    version_added: "2.8"
+  chain:
+    description:
+      - 'The iptables chain in which to insert the direct_rule, used with the C(direct_rule) option.'
```

Comment:
Please do this for all parameters.
```suggestion
      - The iptables chain in which to insert the direct_rule, used with the C(direct_rule) option.
    type: str
```